megatron/model/transformer.py

```diff
@@ -278,17 +278,13 @@ def __init__(
             if self.use_flash_attention:
                 from megatron.model.flash_attention import (
                     flash_attn_unpadded_qkvpacked_func_cuda,
-                    flash_attn_unpadded_qkvpacked_func_triton,
+                    flash_attn_unpadded_kvpacked_func_cuda,
+                    flash_attn_unpadded_unpacked_func_triton,
                 )
 
-                if self.pos_emb == "alibi":
-                    self.flash_attention_function = (
-                        flash_attn_unpadded_qkvpacked_func_triton
-                    )
-                else:
-                    self.flash_attention_function = (
-                        flash_attn_unpadded_qkvpacked_func_cuda
-                    )
+                self.flash_triton_fn = flash_attn_unpadded_unpacked_func_triton
```

Comment:
I believe your branch is based on an older version of gpt-neox. Please rebase your PR to the latest `main` branch

Reply:
sorry about that. currently the polyglot branch maintainer is refactoring the branch for tokenizer inclusion.

Reply:
@chris-ha458 Would it make sense to close this PR and re-open it once the branch has been updated then?